Sources/PrimerSDK/Classes/Data Models/PrimerConfiguration.swift

```diff
@@ -43,6 +43,13 @@ struct PrimerConfiguration: Codable {
             }
         }
         
+#if !canImport(PrimerKlarnaSDK)
+        if let klarnaViewModelIndex = viewModels.firstIndex(where: { $0.config.type == PrimerPaymentMethodType.klarna }) {
+            viewModels.remove(at: klarnaViewModelIndex)
+            print("WARNING!\nKlarna configuration has been found but module 'PrimerKlarnaSDK' is missing. Add `PrimerKlarnaSDK' in your project so you can perform payments with Klarna.")
```

Comment:
Oh, nice 😄 - great minds...
That's exactly the suggestion I've provided in the other PR https://github.com/primer-io/primer-sdk-ios/pull/454#discussion_r955959489 👏